### PR TITLE
finetune map button layout

### PR DESCRIPTION
**Pull request by ztNFny**
fixes https://github.com/cgeo/cgeo/issues/16212

https://github.com/cgeo/cgeo/issues/16212

### CODE REVIEW

**Comment by ztNFny** [comment]
retest this please

File: main/src/main/res/layout/map_zoom_control.xml [hunk 1]
```diff
@@ -7,52 +7,42 @@
...
             android:id="@+id/map_followmylocation_btn"
-            android:tooltipText="@string/menu_centerposition"
-            android:layout_width="24dp"
-            android:layout_height="24dp"
-            android:layout_centerInParent="true"
-            android:contentDescription="@string/menu_centerposition"
-            style="@style/map_quicksettingsbutton" />
+            android:src="@drawable/map_quick_settings"
+            style="@style/map_quicksettingsbutton"
+            android:tooltipText="@string/quick_settings"
```
Comment (moving-bits) [inline main/src/main/res/layout/map_zoom_control.xml:18]: tooltipText and contentDescription should point to "follow my position", currently a long-tap on this icon would show a "quick settings" tooltip

File: main/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -531,6 +531,16 @@
...
     </style>
+    <style name="map_zoombutton">
+        <item name="android:layout_width">38dp</item>
+        <item name="android:layout_height">38dp</item>
+        <item name="android:gravity">bottom</item>
+        <item name="android:insetTop">0dp</item>
+        <item name="android:insetBottom">0dp</item>
+        <item name="android:padding">8dp</item>
+        <item name="android:background">@drawable/mark_transparent</item>
+        <!--<item name="android:tint">@color/osm_zoomcontrol</item>-->
```
Comment (moving-bits) [inline main/src/main/res/values/styles.xml:542]: can be removed (as well as in style above)

**Review by moving-bits — CHANGES_REQUESTED** [review]
Thanks for diving into this, looks much cleaner now!
Have only two small nitpicks, see comments. Thanks!

**Comment by moving-bits** [comment]
One additional question:

Without this PR, the map overlay buttons (zoom controls, quick settings, follow my location etc.) have different levels of transparency/opaqueness:

image

As you can see in the screenshot, zoom buttons are a bit more opaque than map quick settings and route/track quick settings.

With this PR, all buttons will have the higher transparency of those buttons (= less opaqueness), which makes it harder to read the button depending on what beneath them (like the motorway in those screenshots):

image

Can you adjust all map overlay buttons to have the same transparency, based on the more opaque version of the zoom controls in the first screenshot? Thanks.

**Comment by ztNFny** [comment]
The different opaqueness levels were a bug 😉

**Comment by moving-bits** [comment]
> The different opaqueness levels were a bug 😉 Zoom buttons had 2 half-transparent layers on top of each other

Yeah, I know, and it's good that it will be fixed by this PR. Still, the resulting opaqueness of this double-layering was better than the slight opaqueness of the other buttons. So how can we increase the opaqueness without double-layering of images?

**Comment by SammysHP** [comment]
I prefer this version. The buttons can be seen sufficiently and I am still able to guess what's behind them.

**Comment by ztNFny** [comment]
I've adjusted opacity to be between the 2 old ones, don't think anyone would notice the change unless in a direct comparison

**Review by ztNFny — COMMENTED** [review]
I've adjusted opacity to be between the 2 old ones, don't think anyone would notice the change unless in a direct comparison

**Pull request merged**